<launch>
    <!-- <node pkg="pcl_ros" type="pcd_to_pointcloud" name="pcd2pc" args="$(find static_map)/map_file/map_1010_cluster.pcd">
        <remap from="/cloud_pcd" to="/points_raw" />
        <param name="frame_id" value="map" />
    </node> -->
    <node pkg="static_map" type="static_map_node" name="static_map_node" output="screen">
        <remap from="/static_map" to="/map/point_cloud" />
        <!-- <remap from="/initialpose" to="/initial_pose" /> -->
        <remap from="/current_pose" to="/ndt/current_pose" />

        <param name="pcd_file" value="$(find bag_file)/5_voxel1.pcd" />
        <param name="map_frame" value="/map" />
        <param name="duration" value="1.0" />
        <param name="use_local_map" value="true" />
        <param name="local_map_wl" value="20" />
        <param name="local_map_wr" value="20" />
        <param name="local_map_hf" value="40" />
        <param name="local_map_hr" value="20" />
        <param name="local_margin_wl" value="1" />
    </node>
    <!-- <include file="$(find lidar_euclidean_cluster_detect)/launch/lidar_euclidean_cluster_detect.launch" />
    <node pkg="tf" type="static_transform_publisher" name="map2velodyne" args="0 0 0 0 0 0 /map /velodyne 100" /> -->
</launch>